gcloud/bigquery/client.py

```diff
@@ -41,3 +42,14 @@ class Client(JSONClient):
     """
 
     _connection_class = Connection
+
+    def dataset(self, name):
+        """Construct a dataset bound to this client.
+
+        :type name: ``str``
```

Comment:
For native types we have been using words (e.g. integer instead of `int`, string instead of `str`)